arraycontext/context.py

```diff
@@ -463,6 +468,22 @@ def compile(self, f: Callable[..., Any]) -> Callable[..., Any]:
         """
         return f
 
+    def freeze_thaw(self, array: ArrayOrContainerT) -> ArrayOrContainerT:
+        r"""Evaluate an input array or container and return a new array or
+        container representing the evaluation result that is ready for use in
+        arithmetic operations using the execution resources of this context.
+        It does this by calling :meth:`~arraycontext.ArrayContext.freeze` and
+        subsequently :meth:`~arraycontext.ArrayContext.thaw`.
+
+        This method can be used, for example, in the evaluation of any expression
+        trees in the input *array* and producing a 'thaw'\ ed version of the data
+        computed from that evaluation.
+
+        See also :meth:`~arraycontext.ArrayContext.freeze`,
+        :meth:`~arraycontext.ArrayContext.thaw`
```

Comment:
```suggestion
        See also :meth:`freeze` and :meth:`thaw`.
```
? Should work too since they're in the same file.